network-api/networkapi/nav/blocks.py

```diff
@@ -0,0 +1,27 @@
+from wagtail.telepath import register
+
+from networkapi.wagtailpages.pagemodels.customblocks.common.base_link_block import (
+    BaseLinkBlock,
+    BaseLinkBlockAdapter,
+    BaseLinkValue,
+)
+
+
+class NavLinkValue(BaseLinkValue):
+    @property
+    def open_in_new_window(self) -> bool:
+        link_to = self.get("link_to")
+        if link_to == "external_url":
+            return True
+        return False
+
+
+class NavLinkBlock(BaseLinkBlock):
+    class Meta:
+        value_class = NavLinkValue
+        label = "Navigation Link"
+        icon = "link"
+        template = "nav/blocks/nav_link_block.html"
```

Comment:
I can't find the template being referenced. Is that the frontend piece Mavis is working on separately?

Reply:
That's force of habit :see_no_evil: the templates will be worked on later but there's no harm on leaving this here

Reply:
No worries! 😆